bigquery_etl/cli/monitoring.py

```diff
@@ -175,9 +203,15 @@ def validate(name: str, sql_dir: Optional[str], project_id: Optional[str]) -> No
 
     invalid = False
 
-    for bigconfig_file in bigconfig_files:
+    for bigconfig_file in list(set(bigconfig_files)):
         try:
             BigConfig.load(bigconfig_file)
+        except FileLoadException as e:
+            if "Duplicate" in e.message:
+                pass
```

Comment:
Bigeye doesn't like processing the same file twice. It seems to keep a cache of files that it processed, so if this command is called twice on the same file this exception is thrown. We don't want everything to stop, so just ignore duplicate files